nautobot/docs/models/extras/relationship.md

```diff
@@ -22,6 +22,8 @@ Additionally, there are two *symmetric* relationship types that can be used when
 
 Filters can be defined to restrict the type or selection of objects for either side of the connection.
 
+An important note is that the filters have to be defined in **Filterset** Format.
```

Comment:
Perhaps:

```suggestion
An important note is that the filters have to be defined in **FilterSet** rather than QuerySet format. In practice this means that you can use any of the filters that are valid in the REST API for a given object type, but cannot necessarily use complex nested attribute lookups (such as `interfaces__ip_addresses__prefix_length` on a Device, for example).
```